src/components/venues/Jazzbar/components/ReactionList.tsx

```diff
@@ -56,12 +56,8 @@ const ReactionList: React.FC<ReactionListProps> = ({
             <div className="partyname-bubble">
               {usersById[message.created_by].partyName}:
             </div>
-            <div
-              className={`message-bubble ${
-                isMessageToTheBand(message) ? "" : "emoji"
-              }`}
-            >
-              {isMessageToTheBand(message)
+            <div className={`message-bubble ${message ? "" : "emoji"}`}>
```

Comment:
Is this right? How can message be undefined here?

Reply:
Did you mean  `message.reaction === "messageToTheBand"`

Reply:
yes